src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,23 +30,32 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'classy_constructs' => self::NEXT_LINE,
     );
 
     /**
-     * @param array<string, bool>|null $configuration
+     * @param array|null $configuration
+     *
+     * @throws InvalidFixerConfigurationException
      */
     public function configure(array $configuration = null)
     {
-        if (null === $configuration) {
-            $this->configuration = self::$defaultConfiguration;
+        $this->configuration = self::$defaultConfiguration;
```

Comment:
ok. better.
you could patch default config (we usually do it like 
`$configuration = array_merge(self::$defaultConfiguration, $configuration);`
), but not current one.

so, if default is `[foo => 1, bar => 2]`. then
```
$f->configure([foo => 3]); // configured as [foo => 3, bar => 2]
$f->configure([bar => 4]); // configured as [foo => 1, bar => 4], not as [foo => 3, bar => 4]